eng/Subsets.props

```diff
@@ -45,7 +45,7 @@
     <DefaultMonoSubsets Condition="'$(MonoEnableLLVM)' == 'true' and '$(MonoLLVMDir)' == ''">mono.llvm+</DefaultMonoSubsets>
     <DefaultMonoSubsets Condition="'$(MonoAOTEnableLLVM)' == 'true' and '$(MonoAOTLLVMDir)' == ''">mono.llvm+</DefaultMonoSubsets>
     <DefaultMonoSubsets Condition="'$(TargetOS)' == 'Browser'">$(DefaultMonoSubsets)mono.wasmruntime+</DefaultMonoSubsets>
-    <DefaultMonoSubsets Condition="'$(BuildMonoCrossAOT)' == 'true'">$(DefaultMonoSubsets)mono.aotcross+</DefaultMonoSubsets>
+    <DefaultMonoSubsets Condition="'$(BuildMonoCrossAOT)' != ''">$(DefaultMonoSubsets)mono.aotcross+</DefaultMonoSubsets>
```

Comment:
Curious, why don't you check on true not true anymore?

Reply:
It's a string not a bool now. I don't want to test `"+android+browser+" == "true"`, I want to test "Is this not null?"

Reply:
maybe we should rename it to `MonoCrossAOTBuildTargets` to make that clearer?

Reply:
or `MonoCrossAOTTargetOS` like we use in the yaml